src/realm/geospatial.hpp

```diff
@@ -156,7 +156,7 @@ class Geospatial {
     {
     }
     Geospatial(GeoBox box)
-        : m_value(box)
+        : m_value(box.to_polygon())
```

Comment:
It's all unneeded points which stored in Geospatial then if it was constructed from GeoBox. It defeats the purpose of this type a bit, no?

Reply:
I see your point; I have added back support for GeoBox to the Geospatial object.